bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -79,85 +77,85 @@
    * Maximum number of times to retry after a scan timeout
    */
   public static final int DEFAULT_MAX_SCAN_TIMEOUT_RETRIES = 3;
-
+  
+  public static RetryOptions getDefaultOptions() {
+  	return new Builder().build();
```

Comment:
This should probably be `builder().build()` instead of `new Builder().build()`

Reply:
Done.